src/main/resources/application.yml

```diff
@@ -47,7 +49,8 @@ spring:
    rest:
       # this points at the local WireMock server in the test environment.
       iexBasePath: http://localhost:${wiremock.server.port}/
-
+      iexCloudPath: http://localhost:${wiremock.server.port}/
+      iexCloudToken: pk_398f22558d024d7ca05e79961bfdb3f7
```

Comment:
Don't commit tokens in git. Remove this, and wire it in from a file that isn't committed